Doc/library/configparser.rst

```diff
@@ -1043,6 +1043,27 @@ ConfigParser Objects
          config.read(['site.cfg', os.path.expanduser('~/.myapp.cfg')],
                      encoding='cp1250')
 
+      It is possible to read several configurations into a single
+      :class:`ConfigParser`, where the most recently added configuration has the
+      highest priority. Any conflicting keys are taken from the more recent
+      configuration while the previously existing keys are retained.
+
+      .. doctest::
+
+         >>> another_config = configparser.ConfigParser()
+         >>> another_config.read('example.ini')
```

Comment:
There is only one _read_ call, with a single file, so this does not follow the text you added, and seems out of place as an illustration.